x-pack/plugins/telemetry/index.ts

```diff
@@ -33,8 +34,12 @@ export const telemetry = (kibana: any) => {
         config: Joi.string().default(Joi.ref('$defaultConfigPath')),
         url: Joi.when('$dev', {
           is: true,
-          then: Joi.string().default('https://telemetry-staging.elastic.co/xpack/v2/send'),
-          otherwise: Joi.string().default('https://telemetry.elastic.co/xpack/v2/send'),
+          then: Joi.string().default(
+            `https://telemetry-staging.elastic.co/xpack/${ENDPOINT_VERSION}/send`
```

Comment:
should have been more explicit here: I was thinking `xpack/v2/send` should be extracted in a variable.